pkg/benchmarks/browser.js

```diff
@@ -88,7 +96,7 @@ $("#run").click(() => {
             appendResult(
                 "individual",
                 test.title,
-                `https://github.com/peggyjs/peggy/blob/${BRANCH}/benchmark/${benchmark.id}/${test.file}`,
+                `https://github.com/peggyjs/peggy/blob/${BRANCH}/pkg/peggy-benchmarks/${benchmark.id}/${test.file}`,
```

Comment:
i don't understand what this does

Reply:
this is adding links in the HTML so you can see what was being parsed, and is now likely to be incorrect.  Willfix after this lands and I can test it.

Reply:
Again, this is just what was left over from before, it's not like this is code i'm proud of. :)

Reply:
Got it.  [Same issue then](https://github.com/peggyjs/peggy/issues/78)